lib/mayaUsd/nodes/proxyShapeBase.cpp

```diff
@@ -98,6 +98,7 @@ TF_DEFINE_PUBLIC_TOKENS(MayaUsdProxyShapeBaseTokens,
 MayaUsdProxyShapeBase::ClosestPointDelegate
 MayaUsdProxyShapeBase::_sharedClosestPointDelegate = nullptr;
 
+const std::string kAnonymousLayerName{"anonymousLayer1"};
```

Comment:
Not a blocker for the change...but non exported definitions we prefer to put in the unnamed namespace - https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf22-use-an-unnamed-anonymous-namespace-for-all-internalnon-exported-entities. Consider adapting to this rule in a new PR.